### PR TITLE
Fix Bitbank Typos and Missing fiat_ticker doc

### PR DESCRIPTION
**Pull request by macanudo527**
This is to address #128 as well as questions in the wiki about `fiat_ticker`.

### CODE REVIEW

File: docs/configuration_file.md [hunk 1]
```diff
@@ -321,6 +321,7 @@ The `in_csv_file` contains transactions describing crypto being acquired. Line 1
 * `fiat_in_no_fee` (optional): fiat value of the transaction without fee;
 * `fiat_in_with_fee` (optional): fiat value of the transaction with fee;
 * `fiat_fee` (optional): transaction fee (if it was paid in fiat). This is mutually exclusive with `crypto_fee`;
+* `fiat_ticker` (optional): the 3 letter ISO code of the fiat currency being used;
```
Comment (eprbell) [inline docs/configuration_file.md:324]: Let's add a hyperlink to the ISO 4217 Wikipedia page (same thing below):
```suggestion
* `fiat_ticker` (optional): the 3 letter [ISO 4217 code](https://en.wikipedia.org/wiki/ISO_4217#List_of_ISO_4217_currency_codes) of the fiat currency being used;
```

**Review by eprbell — CHANGES_REQUESTED** [review]

**Comment by macanudo527** [comment]
Thanks for the suggestion! I'm sure that will be helpful to users.

File: docs/configuration_file.md [hunk 2]
```diff
@@ -336,6 +337,7 @@ The `out_csv_file` contains transactions describing crypto being disposed of. Li
 * `crypto_out_with_fee` (optional): how much of the given cryptocurrency was sold or sent with the transaction (including fee);
 * `fiat_out_no_fee` (optional): fiat value of the transaction without fee;
 * `fiat_fee` (optional): fiat value of the transaction fee;
+* `fiat_ticker` (optional): the 3 letter ISO code of the fiat currency being used;
```
Comment (eprbell) [inline docs/configuration_file.md:340]: Let's add the link here too.

File: docs/configuration_file.md [hunk 3]
```diff
@@ -349,6 +351,7 @@ The `intra_csv_file` contains transactions describing crypto being moved across
 * `spot_price` (optional): value of 1 unit of the given cryptocurrency at the time the transaction occurred; If the value is unavailable, to direct DaLI to read it from Internet historical data, write in `__unknown` and use the `-s` command line switch;
 * `crypto_sent` (optional): how much of the given cryptocurrency was sent with the transaction;
 * `crypto_received` (optional): how much of the given cryptocurrency was received with the transaction;
+* `fiat_ticker` (optional): the 3 letter ISO code of the fiat currency being used;
```
Comment (eprbell) [inline docs/configuration_file.md:354]: And here.

**Review by eprbell — CHANGES_REQUESTED** [review]

**Comment by macanudo527** [comment]
This should be ok now.

**Review by eprbell — APPROVED** [review]
LGTM

**Pull request merged**